src/main/java/net/glowstone/net/GlowNetworkServer.java

```diff
@@ -8,13 +8,20 @@
 /**
  * Represents a network server.
  *
- * Modified implementation of {@link com.flowpowered.network.NetworkServer}.
+ * <p>Modified implementation of {@link com.flowpowered.network.NetworkServer}.</p>
```

Comment:
Don't use a closing p tag.

Reply:
Fixed. I'll go remove them all on the master branch as well.

Reply:
Sorry for missing those.

Reply:
No, I mean in the branch my other style-fix PRs will be split from. I'm not sure there are any pre-existing closing </p>'s.